lib/widget/answer_multiple_choices.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter_ic/gen/colors.gen.dart';
+import 'package:survey_flutter_ic/model/selection_answer_type_model.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+import 'package:survey_flutter_ic/ui/questions/survey_answer_ui_model.dart';
+
+class AnswerMultipleChoices extends StatefulWidget {
+  final List<SurveyAnswerUiModel> answers;
+  final SelectionAnswerType answerType;
+
+  const AnswerMultipleChoices({
+    super.key,
+    required this.answers,
+    required this.answerType,
+  });
+
+  @override
+  State<AnswerMultipleChoices> createState() => _AnswerMultipleChoicesState();
+}
+
+class _AnswerMultipleChoicesState extends State<AnswerMultipleChoices> {
+  final List<String> _selectedAnswerIds = [];
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        for (int index = 0; index < widget.answers.length; index++) ...[
+          _buildItem(
+            widget.answers[index],
+            _selectedAnswerIds.contains(widget.answers[index].id),
+          ),
+          if (index < widget.answers.length - 1) _buildItemDivider(),
+        ],
```

Comment:
```suggestion
        for (final answer in widget.answers) ...[
          _buildItem(
            answer,
            _selectedAnswerIds.contains(answer.id),
          ),
          if (answer.id != widget.answers.last.id) _buildItemDivider(),
        ],
```
Optional: We can shorten the condition to use `answer` object directly